components/ws-manager/pkg/manager/testdata/status_failedTheiaMount_PENDING00.golden

```diff
@@ -16,7 +16,7 @@
         },
         "phase": 1,
         "conditions": {
-            "failed": "cannot mount Theia",
+            "failed": "MountVolume.SetUp failed for volume \"vol-this-theia\" : hostPath type check failed: /mnt/disks/ssd0/theia/theia-cw-fail-on-mount-failure.2/bogus is not a directory",
```

Comment:
I think we can remove this case alltogether

Reply:
done